### ISSUE TITLE
comma-dangle problem using es6 imports with new line

### ISSUE THREAD

**Issue by bryce-larson** [eslint/eslint#4704]
We are having issues with comma-dangle when are using really long es6 import statements that we wrap with new lines.

eslint: v1.10.3
rule: "comma-dangle": [2, "always-multiline"],
error: Missing trailing comma

```javascript
// error, Missing trailing comma error
import {ReallyLongImportName} from 
  'path/to/really/long/export/file/location/requires/line/wrap'

// works, but line too long
import {ReallyLongImportName} from '../../../path/to/really/long/export/file/location/requires/line/wrap';

// works, but requires comma dangle
import {ReallyLongImportName, } from 
   'path/to/really/long/export/file/location/requires/line/wrap';
```

We would like the first example not to error, since the following works with the same rule:
```javascript
   const {foo} = 
      bar;
```

**Comment by eslintbot** [comment]
@bryce-larson Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by nzakas** [comment]
Can you please provide the requested information?

**Comment by nzakas** [comment]
Confirmed with default parser.

@bryce-larson a quick question: did you see the comment from eslintbot?

**Comment by bryce-larson** [comment]
@nzakas, missed that I am using:
  "parser": "babel-eslint"
